PopPUNK/__main__.py

```diff
@@ -758,10 +832,18 @@ def assign_query(dbFuncs, ref_db, q_files, output, update_db, full_db, distances
             combined_seq, core_distMat, acc_distMat = update_distance_matrices(refList, ref_distMat,
                                                                 ordered_queryList, distMat, query_distMat)
             complete_distMat = translate_distMat(combined_seq, core_distMat, acc_distMat)
+            
+            if assign_lineage:
+                rank_list = sorted([int(x) for x in R.split(',')])
+                expected_lineage_name = ref_db + '/' + ref_db + '_lineageClusters.pkl'
```

Comment:
Would this make sense as a plain text file?

Reply:
Not sure what you mean here - the pkl stores a lot of data.